src/plugins/data/public/antlr/opensearch_ppl/opensearch_ppl_autocomplete.ts

```diff
@@ -11,13 +11,15 @@ import {
   CursorPosition,
   OpenSearchPplAutocompleteResult,
   ProcessVisitedRulesResult,
+  SourceOrTableSuggestion,
 } from '../shared/types';
 import { OpenSearchPPLLexer } from './.generated/OpenSearchPPLLexer';
 import { OpenSearchPPLParser } from './.generated/OpenSearchPPLParser';
 
 // These are keywords that we do not want to show in autocomplete
 export function getIgnoredTokens(): number[] {
-  const tokens = [OpenSearchPPLParser.SPACE, OpenSearchPPLParser.EOF];
+  // const tokens = [OpenSearchPPLParser.SPACE, OpenSearchPPLParser.EOF];
+  const tokens = [];
```

Comment:
do we have an ETA for when we wanted to move this stuff over to monaco package? or maybe even the enhancements plugin?

Reply:
I think after the current set of changes go in. I'm thinking that the likelihood of a breaking change would be high if we attempt to do it now. ETA: Definitely before 2.18

Reply:
Why would this be a breaking change?